### PR TITLE
Feature/105 update documentation

### PR DESCRIPTION
**Pull request by adenh93**

### CODE REVIEW

File: client/src/batching/mod.rs [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+//! batch to ensure that it is delivered to subscribers.
+
+//! # Subscriber
+//!
+//! No stream configuration is required to enable message batching for a
+//! [Publisher](crate::Publisher) stream. As message batches are recieved over the wire in a
+//! [Frame::BatchMessage](selium_protocol::Frame::BatchMessage) frame, the
+//! [Stream](futures::Stream) implementation for the [Publisher](crate::Publisher) stream will
+//! decompress the batch payload (if specified), and then unpack the batch and deliver each message
+//! individually.
```
Comment (ltuch) [inline client/src/batching/mod.rs:35]: As discussed I think the documentation should reference a Subscriber stream

Comment (adenh93) [reply client/src/batching/mod.rs]: Good spot. Resolved this now!

File: standard/src/compression/mod.rs [hunk 2]
```diff
@@ -1,3 +1,203 @@
+//! Implementations for many widely used compression algorithms, including
+//! [DEFLATE](crate::compression::deflate), [lz4](crate::compression::lz4),
+//! [zstd](crate::compression::zstd) and [brotli](crate::compression::brotli).
```
Comment (ltuch) [inline standard/src/compression/mod.rs:3]: Can be addressed later: may be nice to explain the tradeoffs with the different compression algorithms

Comment (adenh93) [reply standard/src/compression/mod.rs:3]: Good point, I may need to do some documentation enhancements next time for the standard crate.

**Review by ltuch — APPROVED** [review]

**Pull request merged**